src/python/ensembl/io/genomio/metadata/update_genome_metadata.py

```diff
@@ -183,7 +180,15 @@ class InputSchema(argschema.ArgSchema):
         dump_default=".",
         metadata={"description": "Directory where the new amended genome file will be created"},
     )
-    brc4_mode = argschema.fields.Int(required=False, metadata={"description": "Activate BRC4 mode (default)"})
+    brc4_mode = argschema.fields.Int(
+        required=False,
+        metadata={"description": "Activate BRC4 mode (default)"}
+    )
+    output_json = argschema.fields.OutputFile(
+        required=False,
+        dump_default="amend_genome_meta.json",
+        metadata={"description": "Default json file to capture json metadata"},
+    )
```

Comment:
Minor: can you run black for this file?

Reply:
Blacked ! Sorry my bad..... I had formatted it to be inline with other option params